docs/docs/tutorials/ais/ais_tutorial.md

```diff
@@ -0,0 +1,572 @@
+
+![Map initialisation](./images/ais_map_visualisation_set.png)
+<p align="center" style="font-style: italic;" >
+Image: Create a visualisation set called Vessel Location
+</p>
+<br />
+
+#### Location layer colored by ship type
+
+Currently, the map is empty. 
+
+Our first goal is to determine where the boats are located and identify their ship types.
+
+![Layer view](./images/ais_layer_view.png)
+<p align="center" style="font-style: italic;" >
+Image: Layer view
+</p>
+<br />
+
+To do so, let's add a layer named `Ship type` to visualise the boats location colored by their type.
+
+In the Geometry section, choose the `point_geom` features geo-field
+
+![Adding a Geometric features layer named 'Boats'](./images/ais_geometric_features_geom.png)
+<p align="center" style="font-style: italic;" >
+Image: Adding a Geometric features layer named 'Ship type'
+</p>
+
+!!! note
+    The layer is included in the visualisation set `Vessels Location`.
+
+<br />
+
+Now, let's define the layer's style. 
+
+As a starter, we choose the best representation of our geometries: Boats positions are points, we represent it as **Circle**. 
+
+We  choose to color the points according to the `Ship type` field values contained in the data.
+
+We fix radius of 4 pixels.
+
+![Customising 'Ship type' style](./images/ais_geometric_features_style.png)
+<p align="center" style="font-style: italic;" >
+Image: Customising the 'Ship type' layer style
+</p>
+<br />
+
+Let's switch to the visibility tab.
+
+We notice that by default, the layer is displayed for all zoom levels (range **[0-23]**) and for a maximum number of elements of **5000**.
+
+If more geopoints are located in your map extend, the layer will no longer be displayed. 
+This limit is often used to switch between direct geometric features layers and aggregated layers (see [Aggregated layer](#aggregated-layer) below). 
+
+Let's increase the limit number of elements at **10000**.
+
+![Customising 'Ship type' visibility](./images/ais_geometric_features_visibility.png)
+<p align="center" style="font-style: italic;" >
+Image: Customising the 'Ship type' layer visibility
+</p>
+<br />
+
+!!! success
+    After clicking on Validate, our first layer is created
+
+![New layer 'Ship type' is created](./images/ais_boats_layer.png)
+<p align="center" style="font-style: italic;" >
+Image:  New layer 'Ship type' is created
+</p>
+
+!!! note
+    We can edit each layer with the Actions button.
+
+<br />
+
+We can now preview the layer in Preview tab
+
+![Preview of 'Boats' layer](./images/ais_boats_layer_preview.png)
+<p align="center" style="font-style: italic;" >
+Image:  Preview of the 'Ship type' layer
+</p>
+<br />
+
+We see now where the boats are passing by thanks to this layer
+<br />
+<br />
+
+#### Geo Big Data: Aggregated Layer
+
+For this tutorial, we only have ~160 000 boats positions to explore.
+
+But what to do in case we had millions of positions to display ?
```

Comment:
Technically we don't need millions of data to ave the issue, just 10000

Reply:
Yes, it was in the previous tutorial.. I changed it